src/global_structs.h

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (C) 2015 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+// This file contains all structs that are needed on both the kernel level and the user level
+// Define eligible structs here instead of importing a .h file into syscall.h
+
+#ifndef GLOBAL_STRUCTS_H
+#define GLOBAL_STRUCTS_H
+
+#include "list.h"
+#include "kerneltypes.h"
+
+struct process_permissions {
+    struct list *directories;
+    int max_number_of_pages;
+    // todo: window stuff
+};
+
+struct process {
```

Comment:
Why is `struct process` declaration in `global_structs.h`? None of the files that included `global_structs.h` (except `process.c` uses `process`